shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -76,238 +72,268 @@
   private static final String ARG_IS_SPLASH_SCREEN_DESIRED = "show_splash_screen";
   private static final String ARG_INITIAL_ROUTE = "initial_route";
   private static final String ARG_APP_BUNDLE_PATH = "app_bundle_path";
+  private static final String ARG_DART_ENTRYPOINT = "dart_entrypoint";
+  private static final String ARG_FLUTTER_INITIALIZATION_ARGS = "initialization_args";
 
   private static final WindowManager.LayoutParams MATCH_PARENT =
       new WindowManager.LayoutParams(WindowManager.LayoutParams.MATCH_PARENT, WindowManager.LayoutParams.MATCH_PARENT);
 
+  /**
+   * Factory method that creates a new {@link FlutterFragment} with a default configuration.
+   *  - no splash screen
+   *  - initial route of "/"
+   *  - default app bundle location
+   *  - default Dart entrypoint of "main"
+   *  - no special engine arguments
+   *
+   * @return new {@link FlutterFragment}
+   */
+  public static FlutterFragment newInstance() {
+    return newInstance(
+        false,
+        null,
+        null,
+        null,
+        null
+    );
+  }
+
+  /**
+   * Factory method that creates a new {@link FlutterFragment} with the given configuration.
+   *
+   * @param isSplashScreenDesired should a splash screen be shown until the 1st Flutter frame is rendered?
+   *                              Defaults to false.
+   * @param initialRoute the first route that a Flutter app will render in this {@link FlutterFragment},
+   *                     defaults to "/"
+   * @param appBundlePath the path to the app bundle which contains the Dart app to execute, defaults
+   *                      to {@link FlutterMain#findAppBundlePath(Context)}
+   * @param dartEntrypoint the name of the initial Dart method to invoke, defaults to "main"
+   * @param flutterShellArgs any special configuration arguments for the Flutter engine
+   *
+   * @return a new {@link FlutterFragment}
+   */
   public static FlutterFragment newInstance(boolean isSplashScreenDesired,
                                             @Nullable String initialRoute,
-                                            @NonNull String appBundlePath) {
+                                            @Nullable String appBundlePath,
+                                            @Nullable String dartEntrypoint,
+                                            @Nullable FlutterShellArgs flutterShellArgs) {
     FlutterFragment frag = new FlutterFragment();
 
+    Bundle args = createArgsBundle(
+        isSplashScreenDesired,
+        initialRoute,
+        appBundlePath,
+        dartEntrypoint,
+        flutterShellArgs
+    );
+    frag.setArguments(args);
+
+    return frag;
+  }
+
+  /**
+   * Creates a {@link Bundle} or arguments that can be used to configure a {@link FlutterFragment}.
```

Comment:
A bundle _of_ arguments?

Reply:
fixed

Reply:
Similarly, this doesn't appear to have been fixed.

Reply:
This change also appears in this PR for me.